compiler-rt/lib/builtins/cpu_model/aarch64/fmv/mrs.inc

```diff
@@ -172,6 +172,11 @@ static void __init_cpu_features_constructor(unsigned long hwcap,
     // ID_AA64ISAR1_EL1.LS64 >= 0b0011
     if (extractBits(ftr, 60, 4) >= 0x3)
       setCPUFeature(FEAT_LS64_ACCDATA);
+
```

Comment:
There is HWCAP2_MOPS support in ELF hwcaps https://docs.kernel.org/arch/arm64/elf_hwcaps.html
which can be also used for mops detection if available

Reply:
Won't be any observable difference as the two shall have the same value.